boards/seeed/xiao_ble/xiao_ble_nrf52840_sense.dts

```diff
@@ -13,6 +13,13 @@
 	model = "Seeed XIAO BLE Sense";
 	compatible = "seeed,xiao-ble", "seeed,xiao-ble-sense";
 
+	msm261d3526hicpm-c-en {
+		compatible = "regulator-fixed";
+		enable-gpios = <&gpio1 10 (NRF_GPIO_DRIVE_S0H1 | GPIO_ACTIVE_HIGH)>;
+		regulator-name = "MSM261D3526HICPM-C-EN";
+		regulator-boot-on;
```

Comment:
i would move this property to the sample overlay, as I don't think users of this board would want the MIC to be always on by default